CMakeLists.txt

```
@@ -23,7 +28,7 @@ target_link_libraries(${PROJECT_NAME} DD4hep::DDCore)
 # Create this_package.sh file, and install
 dd4hep_instantiate_package(${PROJECT_NAME})
 
-install(DIRECTORY compact DESTINATION share/luxegeo/)
+# install(DIRECTORY compact DESTINATION share/luxegeo/)
```

```suggestion
install(DIRECTORY compact DESTINATION share/luxegeo/)
```
I think we would like to keep installing this.